jiant/utils/data_loaders.py

```diff
@@ -286,17 +285,16 @@ def get_tag_list(tag_vocab):
     return tag_list
 
 
-def process_sentence(tokenizer_name, sent, max_seq_len):
-    """process a sentence """
+def truncate_and_tokenize(tokenizer_name, sent, max_seq_len):
+    """Truncate and tokenize a sentence or paragraph."""
     max_seq_len -= 2
-    assert max_seq_len > 0, "Max sequence length should be at least 2!"
+    # TODO: This fn is called potentially millions of times per run.
+    # Can we move the assert elsewhere?
+    assert max_seq_len > 0, "Max sequence length should be at least 2 to allow for boundary tokens!"
```

Comment:
related to TODO: I feel like we really don't need this, but if we did, we can put it at the config check.

Reply:
Deleting assert. Not sure I fully follow the -= 2 problem, but it doesn't seem fatal for BERT, so: Not my monkey; not my circus.